cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/ExecutableBuilder.java

```diff
@@ -366,7 +370,8 @@ private ActionModellingResult compileAction(Map<String, Object> actionRawData) {
 
         String errorMessagePrefix = "Action syntax is illegal.\n";
         actionData.putAll(
-                transformersHandler.runTransformers(actionRawData, actionTransformers, errors, errorMessagePrefix));
+                transformersHandler.runTransformers(actionRawData, actionTransformers, errors, errorMessagePrefix,
+                        SensitivityLevel.ENCRYPTED));
```

Comment:
I see that for executable in/out params you use the param but for step in/params and actions you use hardcoded value? is that the intended behavior?